experiments/dynoptic-ea407ba0a750/build.xml

```diff
@@ -18,118 +18,118 @@
+        <pathelement location="lib/jetty/jsp/jetty-jsp-2.1-7.4.0.v20110414.jar"/>
+        <pathelement location="lib/jetty/jsp/org.apache.jasper.glassfish_2.1.0.v201007080150.jar"/>
+        <pathelement location="lib/jetty/jsp/org.apache.taglibs.standard.glassfish_1.2.0.v201004190952.jar"/>
+        <pathelement location="lib/jetty/jetty-ajp-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-annotations-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-client-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-continuation-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-deploy-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-http-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-io-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-jmx-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-jndi-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-plus-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-policy-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-rewrite-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-security-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-server-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-servlet-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-servlets-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-util-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-webapp-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-websocket-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/jetty-xml-7.3.0.v20110203.jar"/>
+        <pathelement location="lib/jetty/servlet-api-2.5.jar"/>
+        <pathelement location="lib/jung/collections-generic-4.01.jar"/>
+        <pathelement location="lib/jung/jung-algorithms-2.0.1.jar"/>
+        <pathelement location="lib/jung/jung-api-2.0.1.jar"/>
+        <pathelement location="lib/jung/jung-graph-impl-2.0.1.jar"/>
+        <pathelement location="lib/jung/jung-visualization-2.0.1.jar"/>
+        <pathelement location="lib/aspectjrt.jar"/>
+        <pathelement location="lib/automaton.jar"/>
+        <pathelement location="lib/commons-fileupload-1.2.2.jar"/>
+        <pathelement location="lib/commons-io-2.0.1.jar"/>
+        <pathelement location="lib/daikon.jar"/>
+        <pathelement location="lib/derby-10.8.2.2.jar"/>
+        <pathelement location="lib/json-simple-1.1.1.jar"/>
+        <pathelement location="lib/junit-4.9b2.jar"/>
+        <pathelement location="lib/plume.jar"/>
+        <pathelement location="lib/stixar-0.0.1-alpha.jar"/>
     </path>
     <path id="synoptic-ea407ba0a750.classpath">
         <pathelement location="${synoptic-ea407ba0a750.location}/bin"/>
```

Comment:
Instead of copying the jars around, I would recommend that you override $DT_CLASS, $DT_LIBS and the new version's variables after you `source ../setup-vars-ant.sh` in the Synoptic and XML-security files. This way you can sort of special case Synoptic and XML-security's dependencies. 

As of now, your solution requires that you copy around a bunch of Jars and you are still missing dependencies such as `${synoptic-ea407ba0a750.location}/bin`